go.mod

```diff
@@ -63,9 +63,9 @@ require (
 	golang.org/x/exp v0.0.0-20240119083558-1b970713d09a
 	golang.org/x/net v0.24.0
 	golang.org/x/sync v0.6.0
-	google.golang.org/genproto/googleapis/api v0.0.0-20231120223509-83a465c0220f
-	google.golang.org/genproto/googleapis/rpc v0.0.0-20231212172506-995d672761c0
-	google.golang.org/grpc v1.60.1
+	google.golang.org/genproto/googleapis/api v0.0.0-20240304212257-790db918fca8
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240304212257-790db918fca8
+	google.golang.org/grpc v1.62.1
```

Comment:
The upgraded version is still vulnerable to a recently fixed DOS exploit.
https://github.com/grpc/grpc-go/releases/tag/v1.62.2

Reply:
I see. I'll change it to 1.62.2 then, but don't want to address the deprecations with upgrading to 1.64 quite yet. This PR was just to address a unit test failure with the Scheduler branch in isolation.